app/styles/components/events/view/export.scss

```diff
@@ -32,3 +32,11 @@
 .url {
   padding: 0 1em !important;
 }
+
+.copy-url {
```

Comment:
use the `fluid` class instead of this

Reply:
@niranjan94 If I use the class fluid along with the button class, On hover the colors are inverted.
{{#copy-button
    clipboardText=displayUrl
    class='fluid ui secondary segment url button'
  }}
And if I dont use the button class fluid class doesn't work.
This is how it looks on hover : 
![screenshot 2019-02-05 at 9 02 10 pm](https://user-images.githubusercontent.com/31013104/52284044-aeb70880-2989-11e9-965b-e01dba1f4fb2.png)

I haven't made these changes yet. Shall I change it to this?